base-images/build.sh

```diff
@@ -6,12 +6,15 @@ ROOT_DIR=`dirname $0`
 
 GIT_TAG_VERSION=$(git describe --tags --abbrev=0 | sed -r 's/[v]+//g') # remove v from version
 ENVD_VERSION="${ENVD_VERSION:-$GIT_TAG_VERSION}"
+DOCKER_IMAGE_TAG="${DOCKER_IMAGE_TAG:-$ENVD_VERSION}"
 DOCKER_HUB_ORG="${DOCKER_HUB_ORG:-tensorchord}"
 PYTHON_VERSION="${PYTHON_VERSION:-3.9}"
 ENVD_OS="${ENVD_OS:-ubuntu20.04}"
 JULIA_VERSION="${JULIA_VERSION:-1.8rc1}"
 RLANG_VERSION="${RLANG_VERSION:-4.2}"
 
+echo $ENVD_VERSION
+exit 1
```

Comment:
Is this for debug purposes?

Reply:
haha, really sorry for this debug mistake, it must be too late yesterday night and I have been fall asleep 😭

Reply:
:smile: Please update the PR, and, take care.

Reply:
It is a sad story, I tried to adjust my circadian clock, and found out I was a night cat at last🤣

PR updated